plugins/woocommerce/src/Internal/EmailEditor/PersonalizationTagManager.php

```diff
@@ -162,6 +287,43 @@ function ( array $context ): string {
 				},
 			)
 		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Store Name', 'woocommerce' ),
+				'woocommerce/store-name',
+				__( 'Store', 'woocommerce' ),
+				function ( array $context ): string {
+					if ( isset( $context['wc_email'] ) && ! empty( $context['wc_email']->get_from_name() ) ) {
+						return $context['wc_email']->get_from_name();
+					}
+
+					return wp_specialchars_decode( get_bloginfo( 'name' ), ENT_QUOTES );
+				},
+			)
+		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Store Address', 'woocommerce' ),
+				'woocommerce/store-address',
+				__( 'Store', 'woocommerce' ),
+				function (): string {
+					$address_parts = array(
+						WC()->countries->get_base_address(),
+						WC()->countries->get_base_address_2(),
+						WC()->countries->get_base_city(),
+						WC()->countries->get_base_state(),
+						WC()->countries->get_base_postcode(),
+						WC()->countries->countries[ WC()->countries->get_base_country() ] ?? WC()->countries->get_base_country() ?? '',
+					);
+
+					$address = implode( "\n", array_filter( $address_parts ) );
+					return trim( $address );
+				},
```

Comment:
This [method](https://github.com/woocommerce/woocommerce/blob/2afc098512d2f6782339ca0b34c9320c70acdb57/plugins/woocommerce/includes/class-wc-emails.php#L710-L735) does the same thing, so it might be better to reuse it.

Reply:
While implementing the tag, I came across this method, but since it was marked as private, I initially decided not to use it.
After your suggestion, I took a second look I can agree it would be better to reuse it for consistency. I've updated the method visibility to public and refactored the tag to make use of it.